app.js

```diff
@@ -0,0 +1,107 @@
+"use strict";
+
+var restify = require('restify');
+var Sequelize = require('sequelize');
+
+var sequelize = new Sequelize('mvctodo', 'postgres', 'nicopostgres', {
+    host: "localhost",
+    port: 5432,
+    dialect: 'postgres'
+});
+
+var todoentries = sequelize.define('todoentries', {
+    title: {
+        type: Sequelize.STRING,
+        field: 'title'
+    },
+    completed: { type: Sequelize.BOOLEAN }
+},
+    {
+        createdAt: false,
+        updatedAt: false,
+        freezeTableName: true
+    }
+    );
+
+function CreateTodo(req, res, next) {
+    todoentries.sync({ force: true }).then(function () {
+        todoentries.build({
+            title: req.body.title,
+            completed: false
+        })
+            .save()
+            .then(function (test) {
+                res.status(201);
+                res.charSet("utf-8");
+                res.json(test.dataValues);
+            });
+    });
+    next();
+}
```

Comment:
These functions should be defined as closures underneath each restify method definition.